api/src/view_function.rs

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    accept_type::AcceptType,
+    failpoint::fail_point_poem,
+    response::{
+        BadRequestError, BasicErrorWith404, BasicResponse, BasicResponseStatus, BasicResultWith404,
+    },
+    ApiTags, Context,
+};
+use aptos_api_types::{AptosErrorCode, AsConverter, MoveValue, ViewRequest, U64};
+use aptos_vm::AptosVM;
+use move_core_types::language_storage::TypeTag;
+use poem_openapi::{param::Query, payload::Json, OpenApi};
+use std::sync::Arc;
+
+/// API for executing Move view function.
+pub struct ViewFunctionApi {
+    pub context: Arc<Context>,
+}
+
+#[OpenApi]
+impl ViewFunctionApi {
+    /// Execute view function of the module
+    ///
+    /// Execute the Move function with the given paramters and return the execution result.
```

Comment:
Execute the Move function with the given parameters and return its execution result.